recipes/thrift/all/conanfile.py

```diff
@@ -78,7 +78,9 @@ def requirements(self):
             self.requires("libevent/2.1.12")
 
     def build_requirements(self):
-        if tools.os_info.is_windows:
+        build_os = self.settings_build.os if hasattr(self, 'settings_build') else self.settings.os
```

Comment:
```suggestion
        build_os = self.settings_build.os if hasattr(self, "settings_build") else self.settings.os
```

Reply:
I'm used to other languages where single quotes are for literals and double quotes when you want to do some variable processing 😓 . Thanks for the review!

Reply:
The fact that python supports both is a blessing and a curse.